### DISCUSSION TITLE
有人知道更新密钥是个吗啊

### DISCUSSION THREAD

**Discussion by kong-que-dnf123** [Meloong-Git/PCL#3631] [提问] [answered]
我印象中快照版也没要过更新密钥啊.jpg
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/951d51c6-9b57-4579-ad8a-3af20d68cccb)

**Comment by MoYuan-CN** [comment, marked as answer]
在爱发电私信龙猫 `更新 识别码` 获取更新密钥
> 忘了哪个版本在你成功更新之后会保存密钥了

**Comment by wuliaodexiaoluo** [comment]
一段不知道用什么加密算法制作的加密签名
